R/meta_regression/regression_plot_panel.R

```diff
@@ -149,9 +157,13 @@ regression_plot_panel_ui <- function(id) {
 #' @param id ID of the module.
 #' @param model_output GEMTC model results found by calling `CovariateModelOutput()`.
 #' @param treatment_df Reactive containing data frame containing treatment IDs (Number), sanitised names (Label), and original names (RawLabel).
-regression_plot_panel_server <- function(id, model_output, treatment_df) {
+regression_plot_panel_server <- function(id, model_output, treatment_df, reference) {
   shiny::moduleServer(id, function(input, output, session) {
     
+    ref_name <- renderText({
+      reference()
+    })
```

Comment:
`ref_name` is created but never used anywhere? Maybe this was meant to go into the Comparator tooltip?

Reply:
Maybe... I've removed it now because it's not needed.